frontend/src/SinglePlayer/PlayerBanner.jsx

```diff
@@ -1,14 +1,47 @@
 import "./PlayerBanner.css";
 
-function PlayerBanner() {
+function PlayerBanner({ player }) {
+  const playedGames = player[1];
+
   return (
     <div className="playerBanner">
-      <div>
-        <img src="src/assets/movie.png" />
-        <h2> Player Name</h2>
+      <div className="bannerLeft">
+        <h1 className="playerName"> {player[0]}</h1>
+        <h3 className="age">Age: {player[12]}</h3>
```

Comment:
"player[12]" these are pretty ambiguous and hard to read / debug. is there a way to structure this in a way you can read it as a field? i.e. player["age"]

Reply:
Closing pr to update main branch, but will revisit